### DISCUSSION TITLE
Hurco Windows XP controller, connection to MTConnect

### DISCUSSION THREAD

**Discussion by jrozanc007** [mtconnect/cppagent#485] [General]
We are trying to connect our Machine Monitoring systems (3rd part Vimana) to our older Hurco's (VMX24, Running Windows XP). Has anyone  successfully connected MTConnect to older machines using Windows XP? While it looks like the adapter is running, no values are being transferred. I've attached an output from a putty session.

Is anyone aware of a Windows XP or Hurco Winmax setting that may need to be enabled to help with the data transfer?

![agent_output](https://github.com/user-attachments/assets/e879c4ff-6176-48a1-af55-2ea237f1b83e)

**Comment by wsobel** [comment]
I have worked with a number of Hurco machines and you should see if they can update the adapter for you. You should also try to get VIMANA to help with this as well. 

It is probably just the wrong version or an old version of the adapter. Have you spoken with Hurco?

**Reply by jrozanc007** [reply]
Thanks for your insight. Based upon your response it sounds like I shouldn't have an issue with connecting to Windows XP. We have newer Hurco's running Hurco MTConnect 1.25. The only modification we had to take was to disable the agent on the Hurco. Hurco doesn't support dual agent logic (Vimana has their own agent/adapters). I've got Hurco MTConnect versions 1.12 through 1.25. I tried newer versions of MTConnect with Windows XP and found they were incompatible. I'll step up to 1.13, 1.14, etc.. and see if the that fixes our connection issue.

I had our local Hurco tech on site and verify that Winmax/MTConnect was properly configured to communicate. I've found Hurco support/knowledge on their MTConnect integration to be very limited. They know how to install MTConnect by following a small script and that's about it. Troubleshooting help is non-existent. 

Vimana has been helpful but there is only so much they can do remotely. Onsite help of course is T&M.

**Reply by wsobel** [reply]
I wish I could help more. I think you need to contact Hurco and work through their support. Once you get a working adapter, you should be set. 

Best and good luck,
Will Sobel
